iiasa/ixmp
# Add new logo and diagram to Sphinx/RTD docs

This PR adds the new `message_doc` logo to the documentation and is the corresponding PR to https://github.com/iiasa/message_data/pull/368, https://github.com/iiasa/message_ix/pull/597, https://github.com/iiasa/message_doc/pull/53 and https://github.com/iiasa/message-ix-models/pull/68.

## How to review

- Read the diff and note that the CI checks all pass.
- Run a specific code snippet or command and check the output.
- View the RTD docs preview at https://docs.messageix.org/projects/ixmp/en/feature-new-logo/ look at:
  - the combined IIASA-`message-ix-models` logo top left,
  - the favicon
- comment

## PR checklist
